docker/lwua-ingest/lwua-py/lwua/graphdb.py

```diff
@@ -0,0 +1,251 @@
+# this file will be used to trigger the injest processes into graphdb
+# this prevents the injest.py and tryout_watch.py to be run at the same
+# time with conflicting circular imports
+
+from SPARQLWrapper import SPARQLWrapper, JSON
+from datetime import datetime
+from pathlib import Path
+from urllib.parse import unquote, quote
+import logging
+from rdflib import Graph
+import time
+import os
+
+# from dotenv import load_dotenv
+from .helpers import resolve_path  # ,enable_logging
+from pyrdfj2 import J2RDFSyntaxBuilder
+
+log = logging.getLogger(__name__)
+URN_BASE = os.getenv("URN_BASE", "urn:lwua:INGEST")
+
+
+def gdb_from_config():
+    base = os.getenv("GDB_BASE", "http://localhost:7200")
+    repoid = os.getenv("GDB_REPO", "lwua23")
+
+    endpoint = f"{ base }/repositories/{ repoid }"
+    # update statements are handled at other endpoint
+    updateEndpoint = endpoint + "/statements"
+
+    log.debug(f"using endpoint {endpoint}")
+
+    GDB = SPARQLWrapper(
+        endpoint=endpoint,
+        updateEndpoint=updateEndpoint,
+        returnFormat=JSON,
+        agent="lwua-python-sparql-client",
+    )
+    GDB.method = "POST"
+    return GDB
+
+
+GDB = gdb_from_config()
+
+
+def get_j2rdf_builder():
+    template_folder = resolve_path("./lwua/templates")
+    log.info(f"template_folder == {template_folder}")
+    # init J2RDFSyntaxBuilder
+    context = f"{URN_BASE}:ADMIN"
+    j2rdf = J2RDFSyntaxBuilder(
+        templates_folder=template_folder,
+        extra_functions={"registry_of_lastmod_context": context},
```

Comment:
just inline the context variable -- avoids the confusion with the other usage of `context` throughout where it typically does not point to this specific (`{URN_BASE}:ADMIN`) registry_context